includes/class-ed-solr-pagination.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/**
+ * Define the internationalization functionality
```

Comment:
I think this description is not correct.

Reply:
updated